packages/ui/src/pages/AdminPage.vue

```diff
@@ -84,4 +90,13 @@ function handleFileUpload(event: Event) {
       Delete Me! 😲
     </AppButton>
   </div>
+  <div>
+    <ElFormItem label="Email">
+      <ElInput v-model="name" />
+    </ElFormItem>
+
+    <AppButton @click="promoteUser.mutate(name)">
+      Promote another mf to admin🙏
```

Comment:
Since showcase is coming up this weekend, can you make this just say Promote User or something lol

also just change the delete button to say Delete while you're at it (plus anything else like that if it exists)